_layouts/default.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html class="no-js">
   {% include head.html %}
-  <body>
+  <body class="path-support">
```

Comment:
We _could_ but I don't think I actually want to. I wanted to scope the whole site to these styles (since that's where the styles lived in Artoo), but I don't think I actually want the ability/responsibility to have per-article styles.